### PR TITLE
Added feed api

### PR DESCRIPTION
**Pull request by justindhill**
I think I mostly captured the current state of the Feeds.Next API. It's not a perfect clone, but I think that we should work from this and make changes until we're happy with it, then focus on building that thing.

@mariourena @nkranes @omarkarim @nanamkim @DamienBAus @sk8rdave @thenewt15 @sherry-frankly 

These changes are reflected in a rendered format at http://tictocplanet.github.io/feeds-api-docs for easier readability.

### CODE REVIEW

File: swagger.json [hunk 1]
```diff
@@ -2003,6 +2150,342 @@
...
+                "name": {
+                    "description": "The Owner's name.",
+                    "type": "string"
+                },
+                "baseURL": {
+                    "description": "The Owner's base URL.",
+                    "type": "string"
+                },
+                "enableCommentsOnShare": {
+                    "description": "???",
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json [hunk 2]
```diff
@@ -2003,6 +2150,342 @@
...
+                    }
+                },
+                "seo": {
+                    "type": "object",
+                    "$ref": "#/definitions/SEOInfo"
+                },
+                "cls": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "???"
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json [hunk 3]
```diff
@@ -2003,6 +2150,342 @@
...
+                                    "type": "boolean",
+                                    "description": "Whether or not to enable ratings."
+                                }
+                            }
+                        }
+                    }
+                },
+                "sortByDateType": {
+                    "type": "string",
+                    "description": "???",
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json [hunk 4]
```diff
@@ -2003,6 +2150,342 @@
...
+                    "enum": [
+                        "LastEdited"
+                    ]
+                },
+                "implicitContent": {
+                    "type": "boolean",
+                    "description": "???"
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json [hunk 5]
```diff
@@ -2003,6 +2150,342 @@
...
+                },
+                "relatedContentAlignment": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "???"
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json [hunk 6]
```diff
@@ -2003,6 +2150,342 @@
...
+                    "enum": [
+                        "live",
+                        "archived"
+                    ]
+                },
+                "owner": {
+                    "$ref": "#/definitions/Owner"
+                },
+                "ref": {
+                    "description": "???",
```
Comment (justindhill) [inline swagger.json]: Not sure what this field is for - can someone please clarify?

File: swagger.json (hunk 1 above)
Comment (nkranes) [inline swagger.json]: When content is shared across affiliates, this flag sets whether the user wants to allow commenting on that content on the other site.  I believe it displays as a separate commenting section and so affiliates reserve the right to turn it off.

File: swagger.json (hunk 2 above)
Comment (nkranes) [inline swagger.json]: "cls" is the id of the targetadstag item, which is effectively the ad tag.  There are places where we use and id to cross reference this data.

File: swagger.json (hunk 3 above)
Comment (nkranes) [inline swagger.json]: Per item, a user can specify which metadata field to use for sorting, LastEditedDate, PublishedDate, or UpdatedDate.

File: swagger.json (hunk 4 above)
Comment (nkranes) [inline swagger.json]: It allows the user to turn on/off implicit content on the display of this item.  Implicit content is content not directly assigned to the page but assigned via content classification.

File: swagger.json (hunk 5 above)
Comment (nkranes) [inline swagger.json]: "Aligns Story Body Graphics, Ads, Featured Video, Associated Links and the related content to the left or the right of the Story Body Text"

File: swagger.json (hunk 6 above)
Comment (nkranes) [inline swagger.json]: This holds the server and time the page was built from.

Comment (justindhill) [inline swagger.json]: Will this information still be relevant once we are fully on AWS?

File: swagger.json [hunk 7]
```diff
@@ -2003,6 +2150,342 @@
...
+                },
+                "type": {
+                    "type": "string",
+                    "description": "The feature type, i.e. \"story\", \"clip\", \"widget\"."
+                },
+                "headline": {
+                    "type": "string",
+                    "description": "The feature's headline, also sometimes interpreted as its title."
+                },
+                "creationDate": {
```
Comment (justindhill) [inline swagger.json]: @omarkarim I talked to @nkranes the other day about perhaps changing the names of these date fields so their purpose is more clear to the API consumer. He wanted to get your feedback on it, but I'd like to see them changed to:

internalCreationDate
internalLastEditedDate
customerCreationDate
customerLastEditedDate

Thoughts?

File: swagger.json (hunk 4 above)
Comment (justindhill) [inline swagger.json]: Okay, that makes sense. Perhaps we could get rid of this field by not including implicit content for the item if that option is set to `false` on the backend?

File: swagger.json [hunk 8]
```diff
@@ -2003,6 +2150,342 @@
...
+                    "description": "The Owner's id.",
+                    "type": "integer",
+                    "format": "int32"
+                },
+                "name": {
+                    "description": "The Owner's name.",
+                    "type": "string"
+                },
+                "baseURL": {
+                    "description": "The Owner's base URL.",
```
Comment (justindhill) [inline swagger.json]: Just for some clarity - if the feature was owned by WRCB, would this be http://wrcbtv.com?

File: swagger.json (hunk 5 above)
Comment (justindhill) [inline swagger.json]: @DamienBAus @nanamkim is this something you'll be respecting in the new article page on web? If not, perhaps we can remove it from the new feed implementation.

Comment (justindhill) [inline swagger.json]: @mikebachman - perhaps you can weigh in on this, too. Is this something you plan to include in the new designs?

File: swagger.json (hunk 6 above)
Comment (nkranes) [inline swagger.json]: Build server, probably not.  Page generation, possibly.

File: swagger.json (hunk 7 above)
Comment (nkranes) [inline swagger.json]: I was thinking about this and I think that the names have meanings to users in the news business.  Published and Updated, specifically.

File: swagger.json (hunk 8 above)
Comment (nkranes) [inline swagger.json]: Nope.  Just the call letters, wrcb.  They are used for something. Not quite sure what.  @mariourena may have some insight.

File: swagger.json (hunk 4 above)
Comment (nkranes) [inline swagger.json]: I'm not sure I understand.  This data needs to be available in the feed as it changes per Story/Category/Link/Clip and will be necessary for the consumers to know whether they need to make additional calls for implicit content from the server.

File: swagger.json (hunk 8 above)
Comment (justindhill) [inline swagger.json]: Gonna leave this as ??? until we can talk to Mario about what it's used for.

**Pull request merged**